cluster-install/README.adoc

```diff
@@ -74,20 +74,21 @@ The Kops CLI can be used to create a highly available cluster, with multiple mas
 - Setting up AWS resources such as networks, Auto Scaling groups, IAM users, and security groups
 - Installing Kubernetes.
 
-Start the Kubernetes cluster using the following command:
+Create a Kubernetes cluster using the following command. This will create a cluster with a single master, multi node and multi-az configuration:
 
     kops create cluster \
       --name cluster01.kubernetes-aws.io \
```

Comment:
Can we stick to gossip-based cluster, by default? Or add multiple ways to create the cluster in `getting-started` and refer back to that?

It would be better if all the "create cluster" instructions are consolidated into one.